nomad/nomad_node_configs/initial_server.hcl

```diff
@@ -4,6 +4,12 @@ bind_addr = "0.0.0.0"
 server {
   enabled = true
   bootstrap_expect = 1
+  preemption_config {
+    batch_scheduler_enabled    = true
+    system_scheduler_enabled   = true
```

Comment:
Why is this true? We shouldn't need to preempt system jobs right?

Reply:
Preemption of system jobs is enabled by default. Hence, I left it as such. Should we disable it?

Reply:
Ok, if it's on by default that's fine. We also don't have any system jobs, so this won't affect us either way.